src/main/java/com/fismobile/appium/base/BaseClass.java

```diff
@@ -1,4 +1,4 @@
-package com.fismobile.appium.managers;
+package com.fismobile.appium.base;
```

Comment:
Consider renaming the package to a more accurate name. This comment applies for all the packages/classes